.dockerignore

```diff
@@ -0,0 +1,3 @@
+target/
+.github/
+.idea/
```

Comment:
No new line at end of file, please add one :)

Reply:
My bad, new line added!